ams/server/src/main/java/com/netease/arctic/server/ArcticManagementConf.java

```diff
@@ -220,6 +220,18 @@ public class ArcticManagementConf {
           .defaultValue(19090)
           .withDescription("Port that the Http server is bound to.");
 
+  public static final ConfigOption<String> HTTP_SERVER_BASIC_AUTH_USER =
+      ConfigOptions.key("http-server.basic-auth-user")
+          .stringType()
+          .noDefaultValue()
+          .withDescription("The user name of the basic authentication used by REST APIs.");
+
+  public static final ConfigOption<String> HTTP_SERVER_BASIC_AUTH_PASSWORD =
+      ConfigOptions.key("http-server.basic-auth-password")
+          .stringType()
+          .noDefaultValue()
+          .withDescription("The password of the basic authentication used by REST APIs.");
+
```

Comment:
As we already have a `admin-username` and `admin-password`, maybe we could reuse them for REST API authentication.

And you can add a new config `rest-api.auth-type`,  and the value maybe `http-basic-auth` or `token`. the default could be `token`.